src/libs/actions/Policy/Member.ts

```diff
@@ -316,6 +316,16 @@ function removeOptimisticRoomMembers(
             pendingChatMembers: reportMetadata?.pendingChatMembers ?? null,
         },
     });
+    roomMembers.successData.push({
+        onyxMethod: Onyx.METHOD.MERGE,
+        key: `${ONYXKEYS.COLLECTION.REPORT}${report.reportID}`,
+        value: {
+            participants: accountIDs.reduce<Record<number, null>>((acc, curr) => {
+                acc[curr] = null;
+                return acc;
+            }, {}),
+        },
+    });
```

Comment:
NAB. No need to call `roomMembers.successData.push` twice. The line above this one is also `roomMembers.successData.push`. Let's just add this object next to the previous call